tools/cm-package-export-import/export-import-assets.sh

```diff
@@ -0,0 +1,94 @@
+#/*************************************************************************
+#* ADOBE CONFIDENTIAL
+#* ___________________
+#*
+#*  Copyright [first year code created] Adobe Systems Incorporated
```

Comment:
@ankush-mittal [first year code created] have to be replaced with right value.

Reply:
@deepakkr : done